python/scripts/bump.py

```diff
@@ -12,6 +12,8 @@
 
 from semver import VersionInfo
 
+print("This is a TEST PR")
```

Comment:
This test print statement should be removed as it:
1. Could interfere with automated version management processes
2. Adds noise to the script's output which is used for version tracking
3. Doesn't serve any functional purpose in the version bumping logic

If debugging is needed, consider using proper logging with appropriate log levels.